### DISCUSSION TITLE
"Hold Off" mod in mania should get a 0.9x penalty like "Constant Speed"

### DISCUSSION THREAD

**Discussion by Litttlefish** [ppy/osu#21007] [Ideas] [answered]
![图片](https://user-images.githubusercontent.com/38809254/198821330-9cb53731-ebe9-468f-bf62-c43d3d74d6b3.png)
This pic explains why.
A guy used No ln in a ln-dan map

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/pull/21054
